doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -435,64 +439,40 @@ Bluetooth Mesh samples
 
 * :ref:`ble_mesh_dfu_distributor` sample:
 
-  * Added:
-
-    * Support for pairing with display capability and the :ref:`bt_mesh_le_pair_resp_readme`.
-
-  * Fixed:
-
-    * An issue where the shell interface was not accessible over UART because UART was used as a transport for MCUmgr SMP protocol.
-      Shell is now accessible over RTT.
+  * Added support for pairing with display capability and the :ref:`bt_mesh_le_pair_resp_readme`.
+  * Fixed an issue where the shell interface was not accessible over UART because UART was used as a transport for MCUmgr SMP protocol.
+    Shell is now accessible over RTT.
 
 * :ref:`ble_mesh_dfu_target` sample:
 
-  * Added:
-
-    * Support for the :ref:`zephyr:nrf52840dongle_nrf52840`.
+  * Added support for the :ref:`zephyr:nrf52840dongle_nrf52840`.
 
 * :ref:`bluetooth_mesh_light_dim` sample:
 
-  * Added:
-
-    * Support for the :ref:`zephyr:nrf52840dongle_nrf52840`.
-
-  * Fixed:
-
-    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
+  * Added support for the :ref:`zephyr:nrf52840dongle_nrf52840`.
+  * Fixed an issue when building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
 
 * :ref:`bluetooth_mesh_light_lc` sample:
 
-  * Added:
-
-    * Support for the :ref:`zephyr:nrf52840dongle_nrf52840`.
+  * Added support for the :ref:`zephyr:nrf52840dongle_nrf52840`.
 
-  * Fixed:
-
-    * Building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
+  * Fixed an issue when building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards.
```

Comment:
Should this issue be elaborated a bit more to know what the issue is?

Reply:
```
Fixed an issue where Bluetooth couldn't be initialized due to misconfiguration between the Bluetooth host and Bluetooth LE controller when building with :ref:`zephyr:sysbuild` for the :ref:`zephyr:nrf5340dk_nrf5340` and :ref:`zephyr:thingy53_nrf5340` boards
```

Reply:
This need to be changed for all of the places this is written.